Resources/Textures/Objects/Weapons/Guns/Projectiles/particleproj.rsi/meta.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "made by IProduceWidgets (github)",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "particlebolt"
+    },
+	{
+	  "name": "fill1"
+	},
+	{
+	  "name": "fill2"
+	},
+	{
+	  "name": "front"
+	}
+  ]
+}
```

Comment:
This copyright is wrong, you did not make particlebolt it is from tg.

Reply:
Ahh, good catch, originally that rsi only had the fill layers and I ended up changing that. Will adjust.